qpid-jms-client/src/main/java/org/apache/qpid/jms/transports/TransportOptions.java

```diff
@@ -213,6 +215,14 @@ public void setTcpKeepAlive(boolean keepAlive) {
         this.tcpKeepAlive = keepAlive;
     }
 
+    public void setSharedEventLoopThreads(int numThreads) {
+        this.sharedEventLoopThreads = numThreads;
+    }
+
+    public int getSharedEventLoopThreads() {
+        return sharedEventLoopThreads;
+    }
```

Comment:
There are related unit tests for this class and its usage that will need updating. I'll do that as its quicker than explaining.

Reply:
https://github.com/gemmellr/qpid-jms/commit/98904b79e4464aa77c57b062fff858e57983b4dd

Reply:
cherry-picked!